src/code/z_parameter.c

```diff
@@ -3258,9 +3259,9 @@ void Interface_Draw(PlayState* play) {
         }
 
         svar2 = rupeeDigitsFirst[CUR_UPG_VALUE(UPG_WALLET)];
-        svar5 = rupeeDigitsCount[CUR_UPG_VALUE(UPG_WALLET)];
+        svar4 = rupeeDigitsCount[CUR_UPG_VALUE(UPG_WALLET)];
```

Comment:
`Interface_Draw` doesn't match ntsc-1.2

Reply:
Admittedly I blindly trusted `retail_progress.py`, didn't know it could "fail"
I'm curious to know what happens but no it's not worth fixing if it doesm atch (diff.py does indicate so)